test/suite/test_error_info.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import wiredtiger, wttest, time
+from wtdataset import SimpleDataSet
+
+# test_error_info.py
```

Comment:
General comment regarding the names of the test files, should they include a number? We will probably have more than one test file.

Reply:
Would it be ok to leave it as is for now and add numbering after more tests get added?

Reply:
The next person may create a file called `test_error_info01` without realising this one exists. I think it's better to show the way from the start! @jeremythorp, I will let you make the decision here.

Reply:
Thanks @etienneptl for the suggestion. For our Python tests we do generally group them with a name followed by a number, as @etienneptl suggests. However, there are also several Python tests that don't follow that naming convention, and some that have a mixture (eg test_drop.py, test_drop02.py). So I propose we leave it unchanged for now, and add numbering if needed later.

Reply:
Cool, I'll leave it as is and let Etienne confirm/resolve the convo.